python/neuroglancer/pipeline/scripts/ngmesh2obj.py

```diff
@@ -0,0 +1,94 @@
+#!/use/bin/python
+
+from cStringIO import StringIO
+import gzip
+import json
+import os
+import struct
+import sys
+
+from neuroglancer.lib import mkdir
+from neuroglancer.pipeline import CloudVolume, Storage
+
+def download_fragments(cloudpath, mesh_id):
+  vol = CloudVolume(cloudpath)
+  mesh_dir = vol.info['mesh']
+
+  download_path = os.path.join(mesh_dir, mesh_id + ':0')
+
+  with Storage(cloudpath) as stor:
+    fragments = json.loads(stor.get_file(download_path))['fragments']
+    paths = [ os.path.join(mesh_dir, fragment) for fragment in fragments ]
+    frag_datas = stor.get_files(paths)  
+  return frag_datas
+
+def decode_downloaded_data(frag_datas):
+  data = {}
+  for result in frag_datas:
+    data[result['filename']] = decode_mesh_buffer(result['content'])
+  return data
+
+def decode_mesh_buffer(fragment):
```

Comment:
I stole all this stuff from Ricardo and made it slightly more standard for our codebase. 😂